nctl: use new `generic` parameter for global-state-update-gen tool

alsrdn opened the pull request:
Use the `generic` argument for `global-state-update-gen` instead of `change-validators` and `balances`.

See: https://github.com/casper-network/casper-node/issues/3518

alsrdn committed:
nctl: use new `generic` parameter for global-state-update-gen tool

Signed-off-by: Alexandru Sardan <alexandru@casperlabs.io>

fizyk20 commented on utils/nctl/sh/assets/setup_shared.sh:
```diff
@@ -467,15 +474,17 @@ function setup_asset_global_state_toml_for_node() {
                 migrate-into-system-contract-registry -s $STATE_ROOT_HASH -d "$STORAGE_PATH"/"$(get_chain_name)")
 
         GLOBAL_STATE_VALIDATOR_OUTPUT=$("$NCTL_CASPER_HOME"/target/"$NCTL_COMPILE_TARGET"/global-state-update-gen \
-                change-validators $VALIDATOR_SETUP_STRING -s $STATE_ROOT_HASH -d "$STORAGE_PATH"/"$(get_chain_name)") 
+                generic -s $STATE_ROOT_HASH -d "$STORAGE_PATH"/"$(get_chain_name)" $VALIDATOR_SETUP_CONFIG) 
```
The two calls in this file become completely unnecessary now that the `validators` field is optional in the `global_state.toml` file - they were only here so that this field becomes populated. So you should be able to remove them completely, and remove the `validator_setup_config_file` function, too.

alsrdn replied:
Yes, you're right, with the new changes we don't need this call. I'll merge `dev` into this and take them out.

alsrdn committed:
Merge branch 'dev' into nctl-fixes

alsrdn committed:
nctl: remove redundant call to global-state-update-gen

Since the validator list is oprional in the `global_state.toml` file we
can remove the call to `global-state-update-gen` that generates the
validators in `setup_asset_global_state_toml_for_node()`

Signed-off-by: Alexandru Sardan <alexandru@casperlabs.io>

alsrdn commented:
bors merge
